helm/ffc-sfd-file-receiver-infra/templates/namespace-topic.yaml

```diff
@@ -0,0 +1 @@
+{{- include "adp-aso-helm-library.namespace-topic" . -}}
```

Comment:
Alongside declaring the topic/subscription name in the`appConfig` you also need to include ADP's Helm template so that the topic/subscription actually get's created. It goes in the templates folder inside the Infra Helm chart.